src/components/Collapsible/index.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { ReactElement } from 'react';
+import clsx from 'clsx';
+import { Flipped, Flipper } from 'react-flip-toolkit';
+import { Props } from './types';
+
+
+/*
+** Collapsible component uses CSS transitions to smoothly hide and show a block of content.
+*/
+function Collapsible({
+    className,
+    anchorContent,
+    isCollapsed = true,
+    children,
+    ...otherProps
+}: Props): ReactElement {
+
+
+    const [collapsed, setCollapsed] = React.useState(isCollapsed);
+    const wrapperClasses = clsx(
+        'brew-Collapsible',
+        className,
+    );
+    const contentClasses = clsx(
+        {['brew-Collapsible__content']: collapsed},
+        {['brew-Collapsible--open']: !collapsed},
```

Comment:
Could this class go on the parent element instead of the content?

Reply:
Open class sets opacity and transform properties to "unhide" child. We want that hide/show behavior only on the child, while the parent stays static

Reply:
I agree, but the controller styles might change based on whether or not the collapsible is open or not.  That's why I was suggesting that that class be added to the header.

Reply:
I agree. State-based classes should probably always be on the wrapping element. With BEM, it still provides easy styling of children.